pvanalytics/metrics.py

```diff
@@ -1 +1,76 @@
 """Functions for PV system-level metrics."""
+
+from pvlib.temperature import sapm_cell
+from pvlib.pvsystem import pvwatts_dc
+
+
+def performance_ratio_nrel(poa_global, temp_air, wind_speed, pac, pdc0, a=-3.56,
+                          b=-0.075, deltaT=3, gamma_pdc=-0.00433):
+    r'''
+    Calculate NREL Performance Ratio.
```

Comment:
Small style thing - PEP-257 recommends double quotes for docstring delimiters.

```python
    r"""
    Calculate NREL Performance Ratio.
    
    [...]

    """
```

Throughout pvanalytics I have also typically included the summary line on the same line as the delimiter, but PEP-257 says either is okay.